service/src/main/resources/db/migration/V337__duplicate_fee_decision_tables.sql

```diff
@@ -0,0 +1,85 @@
+CREATE TABLE fee_decision_v2 AS SELECT * FROM fee_decision;
```

Comment:
is it fine to copy the data too or should this have something like `WHERE false`?